src/frontend/Felles/Pdf/PdfVisning.tsx

```diff
@@ -3,14 +3,10 @@ import { Ressurs } from '../../App/typer/ressurs';
 import { AlertStripeFeil } from 'nav-frontend-alertstriper';
 import NavFrontendSpinner from 'nav-frontend-spinner';
 import styled from 'styled-components';
-import { Document, Page, pdfjs } from 'react-pdf';
+import { Document, Page } from 'react-pdf/dist/esm/entry.webpack';
 import DataViewer from '../DataViewer/DataViewer';
 import Pagination from 'paginering';
 
-// eslint-disable-next-line
-const pdfjsWorker = require('pdfjs-dist/build/pdf.worker.entry');
-pdfjs.GlobalWorkerOptions.workerSrc = pdfjsWorker;
-
```

Comment:
Den nye importen `import { Document, Page } from 'react-pdf/dist/esm/entry.webpack';` skal gjøre det samme tror jeg.

Reply:
Dobbeltsjekk to ting;
1. At det funker i preprod (ikke bare lokalt - mener det var et problem at workeren ikke ble med i preprod)
2. At størrelsen på bundlen ikke er 10x så stor som før :D

Reply:
Ah, fint du sa i fra. Det virker fint i preprod, men bundlen blir ganske riktig ca. 10x så stor. Får sette workeren manuelt da.

Reply:
Siden vi ikke trenger workeren på alle sidene - så føles det "waste" å skulle ha den i default-bundelen.